Makefile

```diff
@@ -21,27 +30,29 @@ build: vet fmt go_protos swagger_docs
 fmt:
 	@gofmt -l -w $(SRC)
 
-test:
+test: go_protos
 	@go test -v ./...
 
-vet:
+vet: go_protos
 	@go vet -composites=false ./...
 
 protoc/bin/protoc:
 	mkdir -p protoc
 	curl https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-linux-x86_64.zip -o protoc/protoc.zip -L
 	unzip protoc/protoc -d protoc
 
-CLEAN += protoc
+CLEAN += protoc proto/*/*_go_proto
+
+GO_PROTO_DIRS := $(patsubst %.proto,%_go_proto/.done,$(wildcard proto/*/*.proto))
 
-go_protos: grafeas_go_v1alpha1 proto/v1beta1/*_go_proto proto/v1/*_go_proto
+go_protos: v1alpha1/proto/grafeas.pb.go $(GO_PROTO_DIRS)
```

Comment:
What happened to beta and V1?

Reply:
Oh, is it because alpha has a different path format? Can you leave a comment here explaining that?

Reply:
Added the comment, thanks!